ani-cli.1

```diff
@@ -1,138 +1,558 @@
-.TH "ANI-CLI" "1" "January 2023" "ani-cli" "User Commands"
```

Comment:
You replaced the manpage with the script.

Reply:
Yes I replaced